src/utils/ensure-js-imports.test.js

```diff
@@ -85,3 +85,43 @@ test('multiple', () => {
   );
   expect(vars).toEqual([{default: 'foo'}, {default: 'bar', x: 'x'}]);
 });
+
+test('type imports', () => {
+  const path = parseJs(`import A from 'A'`);
+  ensureJsImports(
+    path,
+    `
+    import type {B} from 'A';
+    `
+  );
+  const code = generateJs(path);
+  expect(code.trim()).toMatchInlineSnapshot(
+    `"import A from 'A'import type {B} from 'A';"`
```

Comment:
This looks wrong

Reply:
Pretty sure this is correct... am I missing something?

I guess in theory you could combine them, but that would be a little funky especially when you get into situations where import kinds are mixed.

Reply:
I'm surprised the parser doesn't complain about this, particularly Flow which is finicky about how you describe imports.

For what it's worth, I believe `type {B}` and `{type B}` are interchangeable, and lead to clean merged imports:

```
import A from 'A';
import {type B} from 'A';

=>

import A, {type B} from 'A';
```

Reply:
For the purposes of this codemod, I think it is fine to allow multiple import sources. Collapsing imports is a bit complex, and is left to a separate utility.